charts/guard/values.yaml

```
@@ -0,0 +1,163 @@
+  # as defined in the `httproute-subdomain.yaml` template.
+  # For example, hostnames will be created for `<SERVICE_ID>.localhost`.
+  #
+  # IMPORTANT: For production deployments, the `domain` field should be
+  # overridden with the actual domain name for the PATH service.
+  domain: localhost
+
+  # List of services that will be routed by Envoy Gateway to the PATH backend.
+  # These services will be used to construct HTTPRoutes for each service.
+  # These HTTPRoutes are used to assign the "target-service-id" header to the
+  # request based on the subdomain service alias.
+  #
+  # Configurations:
+  # - serviceId [REQUIRED]: The authorative service ID of the service.
+  # - aliases [OPTIONAL]: A list of aliases for the service.
+  #
+  # For example:
+  # - anvil.localhost -> "target-service-id: anvil"
+  # - F00C.path.grove.city -> "target-service-id: F00C"
+  # - eth.path.grove.city -> "target-service-id: F00C"
+  # - polygon.path.grove.city -> "target-service-id: F021"
+  #
+  # IMPORTANT: For production deployments, the `services` field
+  # should be overridden with the actual services that will be
+  # routed to the PATH backend using a `values.yaml` file.
+  services:
+    # Shannon Example Service ID (with no aliases)
+    - serviceId: anvil
+    # Morse Example Service IDs (with aliases)
+    - serviceId: F00C
+      aliases:
+        - eth
+        - eth-mainnet
+    - serviceId: F021
+      aliases:
+        - polygon
+        - polygon-mainnet
+
+# ---- PATH External Auth Server (PEAS) configuration ----
+authServer:
+  enabled: true
+  replicas: 1
+  port: 10001
+  env:
+    - name: PORT
+      value: "10001"
+    # GRPC_HOST_PORT=guard-pads:10002 is the value to point to the default
+    # PADS server in the cluster created by the GUARD Helm Chart.
+    - name: GRPC_HOST_PORT
+      value: guard-pads:10002
+    # GRPC_USE_INSECURE_CREDENTIALS=true is required to run PEAS in the
+    # cluster created by the GUARD Helm Chart, as PADS does not have TLS
+    # enabled by default.
+    - name: GRPC_USE_INSECURE_CREDENTIALS
+      value: "true"
+
+# ---- PATH Auth Data Server (PADS) configuration ----
+# TODO_MVP(@commoddity): Remove this once Helm Charts Issue #27 is implemented.
+# This will create a simplified default API key implementation for GUARD and remove
+# any dependency on the PADS or PEAS services.
+# See: https://github.com/buildwithgrove/helm-charts/issues/27
+pads:
+  enabled: true
+  replicas: 1
+  port: 10002
+  # IMPORTANT: Exactly one of the following sections below must be set:
+  # - 1. YAML_FILEPATH
+  # - 2. POSTGRES_CONNECTION_STRING
+  # An error will be thrown if neither or both are set.
+
+  # -- 1. YAML_FILEPATH --
+  # IMPORTANT: To use a Postgres database as the PADS data source, comment out
+  # the below section and use the POSTGRES_CONNECTION_STRING section instead.
+  #
+  # If the YAML_FILEPATH environment variable is set,
+  # PADS will use the file at the specified path as the data source.
+
```

Let's put the value from `mountPath` here:

```suggestion

  mountPath: /app/data
```